### PR TITLE
Added Avatar component

### PR DESCRIPTION
**Pull request by rutu14**
https://friendly-kilby-44e498.netlify.app/

image

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,62 @@
+.container{
+    display: flex;
+    justify-content: space-evenly;
+    margin: 2rem;
+}
```
Comment (zoef-yt) [inline components/avatar.css:5]: As I can see you are using this to contain the avatar component but this is not the part component avatar so this line of code should be in a different file.

File: components/avatar.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
```
Comment (zoef-yt) [inline components/avatar.css]: As I can see you are using this to contain the avatar component but this is not the part component avatar so this line of code should be in a different file. Else everything looks good.

Comment (rutu14) [reply components/avatar.css]: Sure.
I missed this , will correct them. 
Thanks

**Review by zoef-yt — APPROVED** [review]

**Review by sanu29 — APPROVED** [review]
Looks good for me. Spacing good in code. Naming convention in continuous and easy class names. Min code in each file. Less repetition of code and file structure is good.
Good to Go

**Review by dev-prabhat — CHANGES_REQUESTED** [review]
Try to use relative unit instead of absolute unit

File: components/avatar.css [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+.avatar-xl{
+    width: 110px;
+    height: 110px;
+}
+
+.avatar-text{
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
```
Comment (dev-prabhat) [inline components/avatar.css:30]: Try to use relative unit instead of absolute unit

File: components/avatar.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    text-transform: uppercase;
+    background-color: var(--quaternary-colour);
+}
+
+.text-sm{
+    font-size: 13px;
+    width: 30px;
+    height: 30px;
```
Comment (dev-prabhat) [inline components/avatar.css]: Try to use relative unit instead of absolute unit

**Review by dev-prabhat — CHANGES_REQUESTED** [review]

File: components/avatar.css (hunk 4 above)
Comment (rutu14) [reply components/avatar.css]: Have changed it today, will update it by tomorrow.

**Pull request merged**